compose/docker-compose.apache.prod.yml

```diff
@@ -30,6 +30,7 @@ services:
       - ADMINMAIL=admin@kimai.local
       - ADMINPASS=changemeplease
       - DATABASE_URL=mysql://kimaiuser:kimaipassword@sqldb/kimai
+      # If you're using a reverse proxy, then add your own domain (example.ltd).
```

Comment:
This is documented in the instructions and listed in the troubleshooting section. If you're adding it here then can you add to the other docker-compose files (including the example in the root of the project)

Reply:
Thanks for quick respond. i will do as you suggested. I also encounter with SMTP configuration. It was well documented in the [docs](https://www.kimai.org/documentation/emails.html#mailer_url). However I was get confused first where should I added with docker method.